### DISCUSSION TITLE
I couldn't hide my file using .gitignore file in GitHub repositories

### DISCUSSION THREAD

**Discussion by manirolo** [community/community#23361] [New to GitHub]
<p>I’m very new to Github. Today I tried to hide a file in my repositories using the <code>.gitignore</code> file. I typed the file name on the <code>.gitignore</code> file. But that file still appears in the repositories.</p>
<p>Can someone help me step by step with how to do it?</p>

**Comment by airtower-luna** [comment]
<p>A <code>.gitignore</code> file does not apply to files that are already part of the index. Could you describe what exactly you are trying to achieve?</p>

**Comment by manirolo** [comment, marked as answer]
<p>Thanks for your support <a class="mention" href="/airtower-luna">@airtower-luna</a>. I created this topic for how to create <code>.gitignore</code> file. Using <code>.gitignore</code>, how to hide a particular file from public view. Now I learned from this video <a href="https://www.youtube.com/watch?v=0WfDe51pUU0&amp;t" class="inline-onebox" rel="noopener nofollow ugc">Part 4 - How to use .gitignore [Git-ing Started with Git Series] - YouTube</a>. I am able to hide the file from public view now.</p>
